ios/RNGeth.swift

```diff
@@ -114,6 +114,85 @@ class RNGeth: RCTEventEmitter, GethNewHeadHandlerProtocol {
         }
     }
 
+    /**
+     * Geth Stats
+     * @return Return Stats from geth
+     */
+    @objc(getGethStats:rejecter:)
+    func getGethStats(resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) -> Void {
+        do {
+            var result = [String: String]()
+            let stats = try runner.getNode().getGethStats()
+            let keys = stats.getStatsKeys()
+            for i in 0..<keys.size()  {
+                let key = keys.get(i)
+                result[key] = stats.getValue(key)
+            }
+
+            resolve([result] as NSObject)
+        } catch let NSErr as NSError {
+            NSLog("@", NSErr)
+            reject(nil, nil, NSErr)
+        }
+    }
+
+    /**
+     * Node Info
+     * @return Return Node info
+     */
+    @objc(getNodeInfo:rejecter:)
+    func getNodeInfo(resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) -> Void {
+        do {
+            var result = [String: String]()
+            let nodeInfo = try runner.getNode().getNodeInfo()
+            result["enode"] = nodeInfo.getEnode()
+            result["id"] = nodeInfo.getID()
+            result["ip"] = nodeInfo.getIP()
+            result["listenerAddress"] = nodeInfo.getListenerAddress()
+            result["name"] = nodeInfo.getName()
+            result["protocols"] = nodeInfo.getProtocols().toString()
+            result["discoveryPort"] = String(nodeInfo.getDiscoveryPort())
+            result["listenerPort"] = String(nodeInfo.getListenerPort())
```

Comment:
nit: you can also define the dictionary directly:
```suggestion
            let nodeInfo = try runner.getNode().getNodeInfo()
            let result: [String: String] = [
                "enode": nodeInfo.getEnode(),
                "id": nodeInfo.getID(),
                "ip": nodeInfo.getIP(),
                "listenerAddress": nodeInfo.getListenerAddress(),
                "name": nodeInfo.getName(),
                "protocols": nodeInfo.getProtocols().toString(),
                "discoveryPort": String(nodeInfo.getDiscoveryPort()),
                "listenerPort": String(nodeInfo.getListenerPort())
            ]
```